src/components/specific/Questions/NavigationButtonInput.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from 'react';
+import CSS from 'csstype';
+import { Select, MenuItem, Checkbox, FormGroup, FormControlLabel } from '@material-ui/core';
+import { useFormikContext } from 'formik';
+import { getPropertyFromDottedString } from '../../../helpers/object';
+import { Form, StepperActions } from '../../../types/FormTypes';
```

Comment:
Use type from helper instead of the StepperAction type, here and in the rest of the file.